### PR TITLE
Update 2021

### PR DESCRIPTION
**Pull request by bogminic**

### CODE REVIEW

**Review by danamunteanu — COMMENTED** [review]
Please review my comments below!

File: README.md [hunk 1]
```diff
@@ -27,6 +27,14 @@ Exercitiile sunt impartite dupa cum urmeaza:
   - [Workshop Angular](doc/workshop-angular-laborator-11-12)
   - [Workshop React](doc/workshop-react-laborator-11-12)
 
+### 🦉 Iconite ajutatoare
```
[the remark this answers is not in the document]
Comment (bogminic) [reply README.md:30]: Done

File: doc/laborator-3/README.MD [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+
+- [🦉 Introducere](#🦉-Introducere)
+- [🚀 Playground](#🚀-Playground)
+- [🎢 Exercitii](##🎢-Exercitii)
+
+## 🦉 Introducere
+
+Ghidul complet de folosire al flexbox-ului il putem gasi pe [CSS tricks](https://css-tricks.com/snippets/css/a-guide-to-flexbox/).
+
+Pe [web.dev](https://web.dev/) putem gasi toate informatiile de baza legate de [responsing web design](https://web.dev/responsive-web-design-basics/).
```
[the remark this answers is not in the document]
Comment (bogminic) [reply doc/laborator-3/README.MD:13]: Done

File: doc/laborator-7/README.MD [hunk 3]
```diff
@@ -141,8 +151,12 @@ Programatorii Javascript obisnuiesc sa foloseasca des expresii precum: “event-
...
+### 🎁 Exercitiul 7.4
 
-### Catelus cu parul cret eXtreme
+#### Catelus cu parul cret eXtreme
 
 - Folositi _[Object literal pattern](https://addyosmani.com/resources/essentialjsdesignpatterns/book/#modulepatternjavascript)_ pentru a rescrie exerciutul anterior.
+
+## 🚀 Playground
+
+[Elevator Saga](https://play.elevatorsaga.com/) ne ajuta sa ne obisnuim cu functiile si evenimentele din Javascript. Pana data viitoare, sa vedem cine rezolva cele mai multe nivele.
```
[the remark this answers is not in the document]
Comment (bogminic) [reply doc/laborator-7/README.MD]: Done

File: doc/laborator-2/README.MD [hunk 4]
```diff
@@ -55,11 +63,15 @@ Iata cum adaugam stiluri paginilor web:
 
 Pentru a invata mai usor proprietatile CSS putem folosi un [Cheat Sheet](css-cheat-sheet.pdf).
 
-## Exercitiul 2.1
+Totodata, ne putem ajuta developer.mozilla.org pentru a aprofunda [selectorii](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors).
```
[the remark this answers is not in the document]
Comment (bogminic) [reply doc/laborator-2/README.MD]: Done

**Review by danamunteanu — APPROVED** [review]

**Review by danamunteanu — APPROVED** [review]

**Pull request merged**